tests/test_porcelain.py

```diff
@@ -1095,6 +1125,28 @@ def test_max_entries(self) -> None:
         porcelain.log(self.repo.path, outstream=outstream, max_entries=1)
         self.assertEqual(1, outstream.getvalue().count("-" * 50))
 
+    def test_no_revisions(self) -> None:
+        outstream = StringIO()
+        porcelain.log(self.repo.path, outstream=outstream)
+        self.assertEqual("", outstream.getvalue())
+
+    def test_empty_message(self) -> None:
+        c1 = make_commit(message="")
```

Comment:
Thanks; I'm not sure why I didn't think to try using an empty string here.  I can confirm that this test fails in the expected manner without the fix applied.